internal/linker/link_node_modules.ts

```diff
@@ -189,37 +207,49 @@ export function main(args: string[], runfiles: Runfiles) {
   }
 
   // Create the $pwd/node_modules directory that node will resolve from
-  symlink(rootDir, 'node_modules');
+  await symlink(rootDir, 'node_modules');
   process.chdir(rootDir);
 
   // Symlinks to packages need to reach back to the workspace/runfiles directory
   const workspaceRelative = path.relative('.', workspaceDir);
   const runfilesRelative = runfiles.dir ? path.relative('.', runfiles.dir) : undefined;
 
   // Now add symlinks to each of our first-party packages so they appear under the node_modules tree
-  for (const m of Object.keys(modules)) {
+  const links = []
+
+  const linkModule = async (name:string,modulePath:string)=>{
     let target: string|undefined;
 
     // Look in the runfiles first
     // TODO: this could be a method in the Runfiles class
     if (runfiles.manifest) {
-      target = runfiles.lookupDirectory(modules[m]);
+      target = runfiles.lookupDirectory(modulePath);
     } else if (runfilesRelative) {
-      target = path.join(runfilesRelative, modules[m]);
+      target = path.join(runfilesRelative, modulePath);
     }
 
     // It sucks that we have to do a FS call here.
     // TODO: could we know which packages are statically linked??
-    if (!target || !fs.existsSync(target)) {
+    if (!target || !await exists(target)) {
       // Try the execroot
-      target = path.join(workspaceRelative, toWorkspaceDir(modules[m]));
+      target = path.join(workspaceRelative, toWorkspaceDir(modulePath));
     }
-    symlink(target, m);
+
+    await symlink(target, name);
   }
 
+  for (const m of Object.keys(modules)) {
+    links.push(linkModule(m,modules[m]))
```

Comment:
hmm where is our linter for missing semicolon?
It matters because clang-format reflows lines and isn't ASI aware